demos/ping_pong.py

```diff
@@ -0,0 +1,233 @@
+# 'Ping pong' interpolation experiment
+# ====================================
+#
+# In this demo, we perform a 'ping poing test', which amounts to interpolating a given
+# sensor function repeatedly between the two meshes using a particular interpolation
+# method. The purpose of this experiment is to assess the properties of different
+# interpolation methods. ::
+
+import os
+
+import matplotlib.pyplot as plt
+from firedrake import *
+from firedrake.pyplot import tricontourf, triplot
+
+from animate import *
+
+# Consider two different meshes of the unit square: mesh A, which is laid out
+# :math:`20\times25` with diagonals from top left to bottom right and mesh B
+# :math:`20\times20` with diagonals from bottom left to top right. ::
```

Comment:
I initially missed that the meshes have different resolutions. Maybe add "a finer mesh A ... and a coarser mesh B"?